lib/http/timeout/global.rb

```diff
@@ -124,7 +126,7 @@ def reset_timer
       def log_time
         @time_left -= (Time.now - @started)
         if time_left <= 0
```

Comment:
Same here. I know that it was before, but please update this one too to use instance variable instead.